### PR TITLE
Simplify tilegrid definition

### PR DESCRIPTION
**Pull request by oterral**
<jenkins>A test link will be added here if the deploy on int succeeded.</jenkins>
This PR gives the same array of resolutions for all the layer WMS and WMTS:
```
[4000, 3750, 3500, 3250, 3000, 2750, 2500, 2250, 2000, 1750, 1500, 1250, 1000, 750, 650, 500, 250, 100, 50, 20, 10, 5, 2.5, 2, 1.5, 1, 0.5, 0.25, 0.1]
```
If a `config.resolutions` array is specified in the layersConfig, we 'll use the last value of the array as minimal resolution for the tile grid, otherwise for WMTS we use the default value specified in the gaGlobalOptions.

<jenkins>[Test link](https://mf-geoadmin3.int.bgdi.ch/teo_tilegrid/index.html)</jenkins>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -10,16 +10,34 @@ define moveto
 	done;
 endef
 
+
+# rc file used
+USER_SOURCE ?= rc_user
+
+
+# Info variables
+USER_NAME ?= $(shell id -un)
```
[the remark this answers is not in the document]
Comment (oterral) [reply Makefile]: Are you sure? for all systems?

**Comment by oterral** [comment]
@gjn @loicgasser @ltclm waht do you think?

File: Makefile [hunk 2]
```diff
@@ -53,47 +75,54 @@ VECTORTILES_URL ?= //vectortiles{s}.geo.admin.ch
...
 DEFAULT_EXTENT ?= '[2420000, 1030000, 2900000, 1350000]'
 DEFAULT_RESOLUTION ?= 500.0
-DEFAULT_LEVEL_OF_DETAIL ?= 7 #level of detail for the default resolution
 RESOLUTIONS ?= '[650.0, 500.0, 250.0, 100.0, 50.0, 20.0, 10.0, 5.0, 2.5, 2.0, 1.0, 0.5, 0.25, 0.1]'
+TILEGRID_ORIGIN ?= '[2420000, 1350000]'
+TILEGRID_RESOLUTIONS ?= '[4000, 3750, 3500, 3250, 3000, 2750, 2500, 2250, 2000, 1750, 1500, 1250, 1000, 750, 650, 500, 250, 100, 50, 20, 10, 5, 2.5, 2, 1.5, 1, 0.5, 0.25, 0.1]'
+
+
+# Globe variables
+DEFAULT_LEVEL_OF_DETAIL ?= 7 #level of detail for the default resolution
```
Comment (procrastinatio) [inline Makefile]: `DEFAULT_LEVEL_OF_DETAIL` is 2500 and  `LEVEL_OF_DETAILS` is 6 from 7/2500 ?

Comment (oterral) [reply Makefile]: default resolution = 500 -> Map zoom = 1 -> Globe lod = 7

**Comment by oterral** [comment]
after  discuss with @loicgasser we should  add a property `tileGridMinResolution` in `layersConfig`.

**Comment by ltclm** [comment]
for type "wmts" layers we have minResolution and maxResolution in the tileset table.
What's the meaning of tileGridMinResolution?
Is it different from minResolution?

**Comment by oterral** [comment]
Yes tileGridMinResolution is the last resolution of the `resolutions` array . We think the resolutions array is useless now.

**Comment by gjn** [comment]
I would avoid putting more in the layersConfig.

As Marcel, I don't understand the difference between minResolution and tileGridMinResolution.

**Comment by oterral** [comment]
omg I didn't see this 1.5 resolution missing

**Comment by oterral** [comment]
@gjn  the resolutions array you mention is the one for the map. This array is still [there](https://github.com/geoadmin/mf-geoadmin3/pull/4273/files#diff-b67911656ef5d18c4ae36cb6741b7965R116)

We talk about removing the resolutions array in layersConfig and add only the last resolution available: 
```
"ch.bafu.neophyten-robinie": {
"opacity": 0.75,
"wmsLayers": "",
"attribution": "OFEV",
"chargeable": false,
"searchable": false,
"format": "png",
"hasLegend": false,
"serverLayerName": "ch.bafu.neophyten-robinie",
"attributionUrl": "https://www.bafu.admin.ch/bafu/fr/home.html",
"tooltip": false,
"label": "Robinier faux acacia",
"highlightable": true,
"background": false,
"timestamps": [
"current"
],
"topics": "api,bafu,ech,inspire,wms-bgdi",
"resolutions": [
4000,
3750,
3500,
3250,
3000,
2750,
2500,
2250,
2000,
1750,
1500,
1250,
1000,
750,
650,
500,
250,
100,
50,
20,
10,
5,
2.5,
2
],
"type": "wmts",
"timeEnabled": false
},
```

@ltclm is ok with that.

**Comment by loicgasser** [comment]
yes this means less info in the layersconfig.

**Comment by oterral** [comment]
Ready for a last review . The change in chsdi can wait.

**Pull request merged**